app/templates/components/custom-smart-banner.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="fandom-app-smart-banner__close" role="button" {{action "close"}}>
```

Comment:
I would rename classes or if you want to have shared styles I would go with more generic class name.